aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -1549,20 +1558,33 @@ std::tuple<Tensor, Tensor> triangular_solve(const Tensor& self, const Tensor& A,
   TORCH_CHECK(self.dim() >= 2,
            "b should have at least 2 dimensions, but has ", self.dim(), " dimensions instead");
   TORCH_CHECK(A.dim() >= 2,
-           "u should have at least 2 dimensions, but has ", A.dim(), " dimensions instead");
+           "A should have at least 2 dimensions, but has ", A.dim(), " dimensions instead");
```

Comment:
Let's get in the habit of specifying the function name at the start of error message with something like "torch.triangular_solve(): ..." or "torch.triangular_solve() expects ...". I realize that PyTorch is inconsistent about this in the current codebase, and doing this isn't a blocker, but I think the error messages are clearer.

Reply:
I will change it to:
```cpp
  TORCH_CHECK(self.dim() >= 2,
           "torch.triangular_solve: Expected b to have at least 2 dimensions, but it has ", self.dim(), " dimensions instead");
  TORCH_CHECK(A.dim() >= 2,
           "torch.triangular_solve: Expected A to have at least 2 dimensions, but it has ", A.dim(), " dimensions instead");
```
It feels awkward to have "self", "input", "other" mixed in code with "A", "b" in error messages. The documented signature for `torch.triangular_solve` is
```python
[torch.triangular_solve(input, A, upper=True, transpose=False, unitriangular=False) -> (Tensor, Tensor)](https://pytorch.org/docs/stable/generated/torch.triangular_solve.html)
```
and then in the text and examples "input" is replaced with "b".

Reply:
You're right. This seems like a good opportunity to consistently refer to the tensors by their names in the signature. NumPy signature names are all over the place, which is not great. Typically in PyTorch we use "input" and "other" for binary operations. Would you like to pick two names and make sure we stick with them? Given the current names, for example, I'd expect the first TORCH_CHECK() in the above snippet to say "Expected input..." instead of "Expected b..." 

Even though "input" and "other" are typical, when there are more natural names, like conventions from the mathematical expressions for an operator, maybe we should use the more natural names, instead? At least for linear algebra, anyway. There's no requirement the first tensor be called "input." 

Also I agree the input vs. self discrepancy is confusing but we can't change that in this PR ;)

Reply:
Current documentation mentions widely used A x = b. I'm more inclined towards using "A", "b" in error messages here as it is user-facing. I think it might be more difficult to remember what "input" and "other" actually refer to in the context of solving matrix equations, especially now when the order of arguments of `torch.linalg.solve`,`torch.linalg.lstsq` is different from `torch.triangular_solve` and other solve functions that are not in `torch.linalg`.

Reply:
SGTM.